pages/profiles/[publicKey]/created.tsx

```diff
@@ -52,6 +59,56 @@ const CreatedNFTs: NextPage<WalletDependantPageProps> = (props) => {
       ? nfts
       : nfts.filter((nft) => nft.name.toLowerCase().includes(query.toLowerCase()));
 
+  const listedNfts = useMemo(
+    () => nftsToShow.filter((nft) => nft.listings.length > 0),
+    [nftsToShow]
+  );
+  const unlistedNfts = useMemo(
+    () => nftsToShow.filter((nft) => nft.listings.length <= 0),
+    [nftsToShow]
+  );
+
+  const totalCount = useMemo(() => nftsToShow.length, [nftsToShow]);
+  const listedCount = useMemo(
+    () => listedNfts?.reduce((acc, nft) => acc + nft.listings.filter((o) => o).length, 0) || 0,
```

Comment:
what does `.filter((o) => o)` accomplish? Wont you get the same result without the filter?

Reply:
checks only for 1+ lengths